go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/ngrok/sqlmw v0.0.0-20200129213757-d5c93a81bec6
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
+	github.com/pressly/goose v2.7.0-rc5+incompatible
```

Comment:
I wonder if we should pin this to a non-release candidate tag.

Reply:
There hasn't been a non RC build in a long time hence why I did this. Some of the examples only work on 2.7